### PR TITLE
[bugfix] Add relation_vns_rs_l_dev_ctx_to_l_dev as Required attribute in resource aci_logical_device_context

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: examples/logical_device_context/main.tf [hunk 1]
```diff
@@ -13,12 +13,17 @@ provider "aci" {
...
-  description       = "from terraform"
-  name_alias        = "example"
+  tenant_dn                          = aci_tenant.example.id
+  ctrct_name_or_lbl                  = "default"
+  graph_name_or_lbl                  = "any"
+  node_name_or_lbl                   = "N1"
+  context                            = "ctx1"
+  description                        = "from terraform"
+  name_alias                         = "example"
+  relation_vns_rs_l_dev_ctx_to_l_dev = "uni/tn-MyTenant01/lDevVip-LoadBalancer01"
```
Comment (lhercot) [inline examples/logical_device_context/main.tf:28]: as your tenant above is not called MyTenant01 you might want to use test_acc_tenant here for the tenant name

File: website/docs/r/function_node.html.markdown [hunk 2]
```diff
@@ -43,7 +43,7 @@ resource "aci_function_node" "example" {
 - `is_copy` - (Optional) If the device is a copy device.
   Allowed values: "yes", "no". Default value: "no".
 - `managed` - (Optional) Specified if the function is using a managed device.
-  Allowed values: "yes", "no". Default value: "yes".
+  Allowed values: "yes", "no". Default value: "no".
```
Comment (lhercot) [inline website/docs/r/function_node.html.markdown:46]: Check on version prior to 5.2 and on 5.2 to confirm behavior.

**Review by lhercot — CHANGES_REQUESTED** [review]

File: examples/logical_device_context/main.tf (hunk 1 above)
Comment (shrsr) [reply examples/logical_device_context/main.tf]: Yeah missed it

File: website/docs/r/function_node.html.markdown (hunk 2 above)
Comment (shrsr) [reply website/docs/r/function_node.html.markdown]: ok

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**